src/Jackett.Common/Definitions/thegeeks.yml

```
@@ -11,87 +11,88 @@ links:
+    - {id: 43, cat: Other, desc: "Comp: Operating Systems"}
+    - {id: 46, cat: Other, desc: "Comp: Security/Encryption"}
     - {id: 52, cat: Other, desc: "Comp: Software Training"}
-    - {id: 202, cat: Other, desc: "Occultism: Mythology Folklore"}
-    - {id: 217, cat: Other, desc: "The Arts: Dance/Singing/Voice"}
-    - {id: 216, cat: Other, desc: "Docu : FlyOnWall Medical"}
-    - {id: 80, cat: Other, desc: "Sports - Live/Highlights"}
     - {id: 41, cat: Other, desc: "Comp: Theory/Ref/Mags"}
-    - {id: 203, cat: Other, desc: "Paranormal"}
-    - {id: 26, cat: Other, desc: "Music: History / Theory"}
-    - {id: 84, cat: Other, desc: "Docu : FrontLine"}
-    - {id: 34, cat: Other, desc: "Docu : Vehicles/Transport"}
     - {id: 47, cat: Other, desc: "Comp: Web Development"}
-    - {id: 204, cat: Other, desc: "Parapsychology"}
-    - {id: 15, cat: Other, desc: "Music: Learning / Courses"}
-    - {id: 36, cat: Other, desc: "Docu : Gardening/Agriculture"}
-    - {id: 207, cat: Other, desc: "Business : Economics"}
     - {id: 68, cat: Other, desc: "Crafts/Jewelry"}
-    - {id: 11, cat: Other, desc: "Political Studies"}
-    - {id: 45, cat: Other, desc: "Photography"}
-    - {id: 58, cat: Other, desc: "Docu : History - Biographies"}
-    - {id: 211, cat: Other, desc: "Business : Investing"}
     - {id: 21, cat: Other, desc: "DIY / Workshop"}
-    - {id: 61, cat: Other, desc: "Science: Biology"}
-    - {id: 55, cat: Other, desc: "Docu : History - Civilization"}
-    - {id: 208, cat: Other, desc: "Business : Marketing"}
     - {id: 25, cat: Other, desc: "Exam Prep / Education"}
-    - {id: 77, cat: Other, desc: "Science: Chemistry"}
-    - {id: 31, cat: Other, desc: "Docu : History - Misc"}
-    - {id: 209, cat: Other, desc: "Business : Startup/Dev"}
     - {id: 9, cat: Other, desc: "Food/Cooking/Nutrition"}
+    - {id: 37, cat: Other, desc: "Game Shows / Quiz Shows"}
+    - {id: 2, cat: Other, desc: "Games: Cards/Tabletop/etc"}
+    - {id: 23, cat: Other, desc: "Hobbies: Misc"}
+    - {id: 1, cat: Other, desc: "Languages / Linguistics"}
+    - {id: 20, cat: Other, desc: "Law & Justice"}
+    - {id: 30, cat: Other, desc: "Literature"}
+    - {id: 16, cat: Other, desc: "Magic & Illusion"}
+    - {id: 206, cat: Other, desc: "Native American Studies"}
+    - {id: 201, cat: Other, desc: "Occultism: Academic / Referenc"}
+    - {id: 210, cat: Other, desc: "Occultism: Bashar"}
+    - {id: 202, cat: Other, desc: "Occultism: Mythology Folklore"}
+    - {id: 202, cat: Other, desc: "Occultism: Mythology Folklore"}
+    - {id: 203, cat: Other, desc: "Paranormal"}
+    - {id: 204, cat: Other, desc: "Parapsychology"}
+    - {id: 11, cat: Other, desc: "Political Studies"}
+    - {id: 61, cat: Other, desc: "Science: Biology"}
+    - {id: 77, cat: Other, desc: "Science: Chemistry"}
     - {id: 12, cat: Other, desc: "Science: Math/Statistics"}
+    - {id: 10, cat: Other, desc: "Science: Medicine/Health "}
+    - {id: 29, cat: Other, desc: "Science: Philosophy"}   
```

remove trailing space to comply with lint
```suggestion
    - {id: 29, cat: Other, desc: "Science: Philosophy"}
```